src/assets/scripts/client/tutorial/TutorialView.js

```diff
@@ -201,8 +201,8 @@ export default class TutorialView {
                 if (prop.aircraft.list.length <= 0) {
                     return t;
                 }
-
-                return t.replace('{CALLSIGN}', prop.aircraft.list[0].callsign);
+                const reverseList = [...prop.aircraft.list].reverse();
```

Comment:
I belive we would still need to reverse it though with ```_filter(aircraftList, (aircraft) => aircraft.flightPhase === FLIGHT_PHASE.APRON)[0]```since as they are loaded in they are added to the end if the array